### PR TITLE
#537 snackbar notification system

### PR DESCRIPTION
**Pull request by magemello**
**Please check if the PR fulfills these requirements**
```
[ ] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
[ ] Tests for the changes have been added (for bug fixes / features)
[ ] Docs have been added / updated (for bug fixes / features)
```

**What kind of change does this PR introduce?** (check one with "x")
```
[ ] Bugfix
[x ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] Documentation
[ ] Other... Please describe:
```

**What is the current behavior?** (You can also link to an open issue here)

#537 

Also added support for Angular/Material 2

**What is the new behavior?**

**Does this PR introduce a breaking change?** (check one with "x")
```
[ ] Yes
[ ] No
```

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: demo-shell-ng2/app/vendor.ts [hunk 1]
```diff
@@ -32,9 +32,13 @@ import 'dialog-polyfill/dialog-polyfill.css';
 
 // Flags
 import 'flag-icon-css/css/flag-icon.min.css';
+
 import '../public/css/app.css';
 import '../public/css/muli-font.css';
 
+// Load the Angular Material 2 stylesheet
+import '../public/css/angular-material2.css';
```
Comment (DenisVuyka) [inline demo-shell-ng2/app/vendor.ts]: We can import this css directly from node_modules

File: demo-shell-ng2/package.json [hunk 2]
```diff
@@ -62,6 +62,7 @@
     "@angular/platform-browser-dynamic": "2.2.2",
     "@angular/router": "3.2.2",
     "@angular/upgrade": "2.2.2",
+    "@angular/material": "^2.0.0-beta.1",
```
Comment (DenisVuyka) [inline demo-shell-ng2/package.json]: I suggest pinning to the specific version. Who knows what they change in the upcoming beta.

File: ng2-components/ng2-alfresco-core/package.json [hunk 3]
```diff
@@ -60,6 +60,7 @@
     "@angular/core": "2.2.2",
     "@angular/forms": "2.2.2",
     "@angular/http": "2.2.2",
+    "@angular/material": "^2.0.0-beta.1",
```
Comment (DenisVuyka) [inline ng2-components/ng2-alfresco-core/package.json]: I suggest pinning to the specific version. Who knows what they change in the upcoming beta.

File: demo-shell-ng2/app/vendor.ts (hunk 1 above)
Comment (DenisVuyka) [inline demo-shell-ng2/app/vendor.ts]: This should work (there are multiple themes available there)
```ts
import '@angular/material/core/theming/prebuilt/deeppurple-amber.css'
```

Comment (magemello) [inline demo-shell-ng2/app/vendor.ts]: thanks

**Pull request merged**